testing/ostinato/APKBUILD

```diff
@@ -0,0 +1,42 @@
+# Maintainer: Corentin Henry <corentinhenry@gmail.com>
+pkgname=ostinato
+pkgver=0.7.1
+pkgrel=0
+pkgdesc="Packet/Traffic Generator and Analyzer"
+url="https://www.ostinato.org/"
+arch="all"
+license="GPLv3"
+depends=
+makedepends="qt-dev protobuf-dev libpcap-dev paxctl"
+subpackages="$pkgname-drone $pkgname-gui"
+source="http://dl.bintray.com/pstavirs/ostinato/ostinato-src-0.7.1.tar.gz"
+
+_builddir="$srcdir"/$pkgname-$pkgver
+
+build() {
+	cd "$_builddir"
+	qmake PREFIX=/usr
```

Comment:
Add `|| return 1` after commands that may fail and should not. This is quite silly, that abuild doesn’t run in `set -e` mode, but it’s already planned to fix it.

```diff
- qmake PREFIX=/usr
+ qmake PREFIX=/usr || return 1
make
```